src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -52,6 +52,31 @@ export default function Analysis() {
     AnalysisContext
   );
 
```

Comment:
I a wondering if the code you have 

```javascript
// Built in Analysis Types
const analysisTypesWithAdditionalPage = [
  "BIO_HANSEL",
  "SISTR_TYPING",
  "PHYLOGENOMICS",
  "MLST_MENTALIST"
];
```
Should not be given dynamically from the server.

Reply:
Yeah that's the plan once I get to that part